conftest.py

```diff
@@ -82,6 +82,51 @@ def res_dec(res_path, misc_info):
     return res_path / (misc_info.FNames.RES.value + misc_info.Extensions.DEC.value)
 
 
+@pytest.fixture(scope="session")
+def res_cmp_plus_one_line(res_path, misc_info):
```

Comment:
Is this fixture needed any more, since this PR will only be adding the `sphobjinv-textconv` endpoint and not actually doing anything with git diffs?